ZenPacks/zenoss/Microsoft/Windows/resources/js/global.js

```diff
@@ -80,7 +87,7 @@ Ext.define("Zenoss.component.WinRMServicePanel", {
     }
 });
 
-ZC.registerName('WinRMService', _t('Service'), _t('Services'));
```

Comment:
instead of making changes to this js, let's just use the existing one in platform.  doing this stuff makes it look just like that one anyway